pkl-core/src/main/resources/org/pkl/core/errorMessages.properties

```diff
@@ -797,7 +800,8 @@ Modifier `const` cannot be applied to property `{0}`.\n\
 Property `{0}` cannot be declared const, because it overrides a non-const property on parent class `{1}`.
 
 unseparatedObjectMembers=\
-Object members must be separated by whitespace, newline, or semicolon.
+Object properties and elements must be separated by whitespace, newline, or semicolon.\n\
```

Comment:
There's also for generators, spreads, when, etc.

I think `Object members` is better. We only present this error message in the case of object members that we can successfully parse on the same line.